runall.sh

```diff
@@ -7,6 +7,7 @@ root=`realpath \`dirname $0\``
 BUILD=$root/build
 OUT=$BUILD/out
 test -d bundler-spec-tests || git clone https://github.com/alchemyplatform/bundler-spec-tests.git
+git -C bundler-spec-tests checkout alex/revert-gas-changes
```

Comment:
can remove this once https://github.com/alchemyplatform/bundler-spec-tests/pull/2 gets merged